packages/dart/lib/src/base/parse_constants.dart

```diff
@@ -1,7 +1,7 @@
 part of flutter_parse_sdk;
 
 // Library
-const String keySdkVersion = '3.1.2';
+const String keySdkVersion = '3.1.4';
```

Comment:
Did you manually change this version or is this auto-generated?

Reply:
> Did you manually change this version or is this auto-generated?

manually

Reply:
We haven't changed that in the last release, maybe by mistake?

I think we should create a list where the version has to be changed for a new release.

Reply:
> We haven't changed that in the last release, maybe by mistake?

Yes, in the previous version, I didn't know exactly that I had to change the parse_constants file

> I think we should create a list where the version has to be changed for a new release. So for dart it's in:
> * packages/dart/lib/src/base/parse_constants.dart
> * packages/dart/pubspec.yaml
> 
> ?

Yes, it's a good idea

Reply:
Why is this variable here needed at all?

Reply:
This variable is sent to the server as the version of the requester in a header in keys ```parseVersion``` and ```user-agent```

and It is not a critical variable, but it is good to change it

Reply:
So this was actually bug in the previous version then. Is there any way we could get the package version from the package itself instead of manually maintaining this var?

Reply:
> So this was actually bug in the previous version then. 

yes
> Is there any way we could get the package version from the package itself instead of manually maintaining this var?

Yes, but you have to use other packages [package_info_plus](https://pub.dev/packages/package_info_plus) and this package does not support the Dart version, it only supports Flutter!

Reply:
@mtrezza 
Is it possible to Merge this PR? I need these codes in another PRs.

Reply:
Sure, let' keep this in mind, maybe we can find a way to replace that extra var in the future...